trendshop/trendshop/settings.py

```diff
@@ -89,8 +90,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME':'trendshop',
+        'USER':'postgres',
+        'PASSWORD':'postgres',
+        'HOST':'localhost',
+        'PORT':5432,
     }
```

Comment:
next time for creating a project use cookiecutter and add database config in dev.py.

Reply:
okk

Reply:
what's cookiecutter? do you have any doc or video to learn and know about it